accept_test.go

```
@@ -425,3 +425,107 @@ func BenchmarkAcceptHeader(b *testing.B) {
+			patterns: []string{
+				"https://*.example.com",
+			},
+			success: false,
+		},
+	}
+
+	for _, tc := range testCases {
+		t.Run(tc.name, func(t *testing.T) {
+			t.Parallel()
+
+			r := httptest.NewRequest(http.MethodGet, "http://"+tc.host, nil)
```

golangci-lint (2.12.2)

[error] net/http/httptest.NewRequest must not be called. use net/http/httptest.NewRequestWithContext

(noctx)